tests_integration/test_external_plugins.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from tests_integration.testing_utils.assertions.test_result_assertions import (
+    assert_that_result_is_successful,
+    assert_that_result_is_successful_and_output_json_contains,
+)
+from tests_integration.snowflake_connector import snowflake_session
+
+
+@pytest.mark.integration
+def test_loading_of_installed_plugins_if_all_plugins_enabled(runner, snowflake_session):
+    runner.use_config("config_with_enabled_all_external_plugins.toml")
```

Comment:
Maybe instead of overriding default config, create new `invoke` where you can pass path to config?

Reply:
I started with that approach but then I noticed that I am limited to only one version of `invoke` and I have to duplicate code.

The way implemented it allows me to use every version of `invoke` and makes the usage less complitated.